plugins/inputs/uwsgi/uwsgi.go

```diff
@@ -0,0 +1,181 @@
+package uwsgi
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/plugins/inputs"
+	"io"
+	"net"
+	"net/http"
+	"net/url"
+	"strconv"
+	"time"
+)
+
+var timeout = 5 * time.Second
+
+type Uwsgi struct {
+	Servers []string `toml:"server"`
+}
+
+func (u *Uwsgi) Description() string {
+	return "Read uWSGI metrics."
+}
+
+func (u *Uwsgi) SampleConfig() string {
+	return `
+    ## List with urls of uWSGI Stats servers. Url must match pattern:
+    ## scheme://address[:port]
+    ##
+    ## For example:
+    ## servers = ["tcp://localhost:5050", "http://localhost:1717", "unix:///tmp/statsock"]
+    servers = []
+`
+}
+
+func (u *Uwsgi) Gather(acc telegraf.Accumulator) error {
+	for _, s := range u.Servers {
+		n, err := url.Parse(s)
+		if err != nil {
+			return fmt.Errorf("Could not parse uWSGI Stats Server url '%s': %s", s, err)
+		}
+
+		u.gatherServer(acc, n)
```

Comment:
you shouldn't ignore the error here, you should accumulate any errors from servers that fail and then return that.